src/sentry/static/sentry/app/views/releases/utils/sessionTerm.tsx

```diff
@@ -28,6 +29,7 @@ export const commonTermsDescription = {
   [SessionTerm.CRASHES]: t('Number of sessions with a crashed state'),
   [SessionTerm.CRASH_FREE_USERS]: t('Number of unique users with non-crashed sessions'),
   [SessionTerm.CRASH_FREE_SESSIONS]: t('Number of non-crashed sessions'),
+  [SessionTerm.STABILITY]: t('Stability is the percentage of crash free sessions.'),
```

Comment:
should we remove the punctuation here?

Reply:
I wanted to be consistent as other terms on that page (coming from performance) have dots